docs/topics/Gateway.md

```diff
@@ -9,8 +9,9 @@ Important note: Not all event fields are documented, in particular, fields prefi
 ###### Gateway Versions
 
 | Version | Status       |
-| ------- | ------------ |
-| 6       | Available    |
+|---------|--------------|
+| 8       | Available    |
```

Comment:
You documented (or...uh...mentioned) v7 for REST, should you mention v7 here too?